<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImpellerWidget</class>
 <widget class="QWidget" name="ImpellerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="50,25,25">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="70,5,17,8">
     <item>
      <widget class="QLabel" name="impellerImageLabel">
       <property name="styleSheet">
        <string notr="true">image: url(:/impeller/images/imp.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Report of Calculation:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit">
       <property name="styleSheet">
        <string notr="true">QTextEdit {
	border-color: rgb(189, 255, 148);
}

QTextEdit:pressed {
	border-color: rgb(255, 247, 164);
}</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="50,50">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="saveReportPushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>50</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: rgb(55, 179, 74);
	color: rgb(255, 255, 255);
	border: none;
	border-radius: 10px
}

QPushButton:hover {
	background-color: rgb(71, 226, 90);
}

QPushButton:pressed {
	background-color: rgb(108, 244, 121);
}</string>
         </property>
         <property name="text">
          <string>Save Report</string>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>253</width>
        <height>648</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Pump Parameters:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Volume Flow Rate, Q [m&lt;span style=&quot; vertical-align:super;&quot;&gt;3&lt;/span&gt;/h]&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="volumeFlowRateDoubleSpinBox">
         <property name="maximum">
          <double>10000000.000000000000000</double>
         </property>
         <property name="value">
          <double>50.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Head of Pump, H [m]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="headPumpDoubleSpinBox">
         <property name="maximum">
          <double>1000000.000000000000000</double>
         </property>
         <property name="value">
          <double>20.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Rotation Speed, n [rpm]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="rotationSpeedDoubleSpinBox">
         <property name="maximum">
          <double>1000000.000000000000000</double>
         </property>
         <property name="value">
          <double>3000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Number of Stages, i [-]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="numberStagesSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Type of Pump</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="typePumpComboBox">
         <item>
          <property name="text">
           <string>Single Stage, Singe Entry</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Single Stage, Double Entry</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Multistage, Single Entry</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Shaft Safety Factor, s&lt;span style=&quot; vertical-align:sub;&quot;&gt;st&lt;/span&gt; [-]&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="shaftSafetyFactorDoubleSpinBox">
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
         <property name="value">
          <double>1.100000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;H-Q Curve, f&lt;span style=&quot; vertical-align:sub;&quot;&gt;t&lt;/span&gt; [-]: &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="hQCurveDoubleSpinBox">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1.100000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.001000000000000</double>
         </property>
         <property name="stepType">
          <enum>QAbstractSpinBox::DefaultStepType</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_17">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Blades Number, Z [-]:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSpinBox" name="bladesNumberSpinBox">
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>12</number>
         </property>
         <property name="value">
          <number>7</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Trailling Edge, β&lt;span style=&quot; vertical-align:sub;&quot;&gt;2&lt;/span&gt; [deg]:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox">
         <property name="minimum">
          <double>10.000000000000000</double>
         </property>
         <property name="maximum">
          <double>50.000000000000000</double>
         </property>
         <property name="value">
          <double>23.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="92,8">
     <item>
      <widget class="QScrollArea" name="scrollArea_2">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>250</width>
          <height>588</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Specific Speeds:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout" columnstretch="25,75">
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>Calibri</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;n&lt;/span&gt;&lt;span style=&quot; font-size:9pt; vertical-align:sub;&quot;&gt;s &lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;(RU):&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <family>Calibri</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;n&lt;span style=&quot; vertical-align:sub;&quot;&gt;q&lt;/span&gt; (EN):&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="nsLabel">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="nqLabel">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Meridional Dimensions:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_2" columnstretch="25,75">
           <item row="0" column="1">
            <widget class="QLabel" name="d2Label">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_10">
             <property name="font">
              <font>
               <family>Calibri</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;D&lt;span style=&quot; vertical-align:sub;&quot;&gt;2&lt;/span&gt; [mm]:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_11">
             <property name="font">
              <font>
               <family>Calibri</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;b&lt;span style=&quot; vertical-align:sub;&quot;&gt;2&lt;/span&gt; [mm]:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="b2Label">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="calculatePushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>50</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: rgb(55, 179, 74);
	color: rgb(255, 255, 255);
	border: none;
	border-radius: 10px
}

QPushButton:hover {
	background-color: rgb(71, 226, 90);
}

QPushButton:pressed {
	background-color: rgb(108, 244, 121);
}</string>
         </property>
         <property name="text">
          <string>Calculate</string>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../styles/impellerwidget.qrc"/>
 </resources>
 <connections/>
</ui>
